docs-md/users/quickstart.md

```diff
@@ -98,6 +98,12 @@ The pairing information is persisted on your Ambianic UI client device and you c
 
 When you are ready to explore more advanced capabilities, continue to the next section.
 
+### Ambianic Premium Services
```

Comment:
@vickywane I thought I made this comment yesterday, but I don't see it in the comment log, so here it is again. Please update the order of sections. Edge Pairing is a more essential step that should come before Premium Services. Let's position Premium Services right before Configuration. Also let's make a small language adjustment. See below:

```

### Ambianic Premium Services

Ambianic provides premium value added services such as email notifications for users who prefer the convenience of automatic integration with third party cloud services. 

Read on: [Using Ambianic Premium Services](premium-services.md)

### Configuration

....

```

Reply:
Alright. 

I have made this adjustment.